docs/operations/records-management.md

```diff
@@ -0,0 +1,7 @@
+---
+id: records-management
+title: Considerations and Guidance for Records Management regards Open Source
+sidebar_label: Record Management Policies
+---
+
+Financial companies do define their retention policies when comes to record management, like how email communication, internal source code, internal chat systems (all involving possible counter parties) are applied to all firm business; some of these even surveiled in real time. However, such policies only do apply to what is considered firm business. As by definiton the Open Source work is not considered a firm business, only the firm wide communication policies that would be applied to the incoming email notifications from GitHub, would be applicable here.
```

Comment:
* Remove the 'do' here.
* Link "open source work is not considered a firm business" to the Electronic Communications Policies resource page
* Add another paragraph advising people to double check the policies with their lawyers/etc.
* Change to read "Open source work is not considered firm-related business"

Reply:
@vmbrasseur please recheck; I think I covered everything from the meeting